packages/snap/@types/lotus-client-schema/index.d.ts

```diff
@@ -1,12 +1,519 @@
-declare module "@filecoin-shipyard/lotus-client-schema" {
-  export type Schema = {
-    methods: any;
-  };
```

Comment:
This is an override they use to ignore the types from  the lotus-client-schema. Removing this file should move typescript to the npm types directly.